website/content/api-docs/system/replication/index.mdx

```diff
@@ -197,3 +199,69 @@ performance secondary and DR primary node, it will look something like:
   }
 }
 ```
+
+## Check Merkle Tree Corruption
+
+This endpoint prints information about the corruption status of the merkle tree.
+This endpoint would offer a certain level of assurance that issues with replication
+can be resolved through reindexing. This is desirable in particular since reindexing
+can be time-consuming and lead to downtime, which may not be appealing to customers.
+Thus, the endpoint serves as a valuable aid in helping customers decide whether a
+reindex is necessary.
+
+This is an authenticated endpoint.
```

Comment:
What does "this is an authenticated endpoint" mean?

Reply:
It means that it needs a token.

Reply:
Understood. Added a suggestion to clarify things for readers who may be new to Vault.